manifests/init.pp

```diff
@@ -177,6 +177,10 @@
 # @param [Boolean] auto_ruleset_updates
 #    Enable automatic rule updates?
 #
+# @param [Boolean] manage_repo
```

Comment:
please don't add the datatype here. that's a legacy puppet-strings style. puppet-strings is able to read it from the declared parameter in line 238
```suggestion
# @param manage_repo
```